src/network.rs

```diff
@@ -1,37 +1,35 @@
-use crate::{
-    bitcoin,
-    ethereum::{self, dai},
-    order::{BtcDaiOrder, Position},
-    swap::{hbit, herc20, SwapKind, SwapParams},
-    Seed,
-};
+use crate::{bitcoin, ethereum, swap::SwapKind, Seed, SwapId};
 use ::bitcoin::hashes::{sha256, Hash, HashEngine};
-use chrono::Utc;
-use comit::{
-    asset, identity,
-    network::{
-        self,
-        orderbook::{self, take_order, OrderId},
-        Comit, Identities, LocalData, Orderbook, RemoteData,
-    },
-    SharedSwapId, Timestamp,
-};
+use comit::{identity, network::orderbook, BtcDaiOrder, Position, Role, Secret, SecretHash};
 use futures::Future;
 use libp2p::{
     identity::ed25519,
-    request_response::ResponseChannel,
-    swarm::{NetworkBehaviourAction, NetworkBehaviourEventProcess, PollParameters},
+    identity::Keypair,
+    swarm::{NetworkBehaviourAction, PollParameters},
     NetworkBehaviour, PeerId,
 };
 use serde::{de::Error, Deserialize, Deserializer, Serialize, Serializer};
 use std::{
-    collections::{HashMap, VecDeque},
+    collections::VecDeque,
     pin::Pin,
     str::FromStr,
     task::{Context, Poll},
 };
 
-use crate::swap::Database;
+use crate::swap::SwapParams;
+
+use chrono::Utc;
+
```

Comment:
Not sure what nectar policy is here but your imports are scattered all over and hence not formatted :)

Reply:
there doesnt seem to be any auto formatting, I don't think it is worth the effort fixing this manually at the moment